### PR TITLE
New package: Zen-Team.Zen-Browser version 1.0.0-a.28

### PR DESCRIPTION
**Pull request by sitiom**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/168940)

### CODE REVIEW

**Comment by UnownPlain** [comment]
This package already exists (#168542). You should update the existing one.

**Comment by sitiom** [comment]
My bad. To be fair, Komac, @wingetbot, and a quick search of "Zen Browser" did not yield any results 😅

File: manifests/z/Zen-Team/Zen-Browser/1.0.0-a.28/Zen-Team.Zen-Browser.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Created with komac v2.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Zen-Team.Zen-Browser
+PackageVersion: 1.0.0-a.28
+PackageLocale: en-US
+Publisher: Mozilla
```
Comment (sitiom) [inline manifests/z/Zen-Team/Zen-Browser/1.0.0-a.28/Zen-Team.Zen-Browser.locale.en-US.yaml:7]: - https://github.com/zen-browser/desktop/issues/483

**Comment by UnownPlain** [comment]
It looks like this package installs the generic version while the optimized version is a separate package (#168942). I think it makes more sense to have this be the optimized version and the generic version be in a separate package since most systems should support the faster optimized version.

See: https://docs.zen-browser.app/guides/generic-optimized

The optimized version is supported in all systems with an Intel 4th Gen+ CPU and all Ryzen CPUs.

**Comment by sitiom** [comment]
> It looks like this package installs the generic version while the optimized version is a separate package (#168942). I think it makes more sense to have this be the optimized version and the generic version be in a separate package since most systems should support the faster optimized version.
> 
> See: [docs.zen-browser.app/guides/generic-optimized](https://docs.zen-browser.app/guides/generic-optimized)
> 
> The optimized version is supported in all systems with an Intel 4th Gen+ CPU and all Ryzen CPUs.

I'm following the AUR here ([`zen-browser-bin`](https://aur.archlinux.org/packages/zen-browser-bin) and [`zen-browser-optimized-bin`](https://aur.archlinux.org/packages/zen-browser-optimized-bin)), which makes sense since the "optimized" is the "specific build" and the main package is expected to work for all. I still even have a laptop that doesn't support the "optimized" build. @mauro-balades Any thoughts here?

**Review by stephengillie — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**